be/src/exec/vectorized/orc_scanner_adapter.cpp

```diff
@@ -1169,6 +1169,19 @@ Status OrcScannerAdapter::_init_position_in_orc() {
         SlotId id = slot_desc->id();
         _slot_id_to_position[id] = pos;
     }
+
+    if (_lazy_load_ctx != nullptr) {
+        for (int i = 0; i < _lazy_load_ctx->active_load_slots.size(); i++) {
+            int src_index = _lazy_load_ctx->active_load_indices[i];
+            int pos = _position_in_orc[src_index];
+            _lazy_load_ctx->active_load_orc_positions[i] = pos;
```

Comment:
the motivation of former emplace(0) is make room for here?

Reply:
Right.